src/Players/Player.cpp

```diff
@@ -1,43 +1,42 @@
 #include "./Player.h"
 #include "../Strategist/Strategist.h"
 
-//Auto assigns this player's race and sets enemies race to unknown
+#include <limits>
+
+#include "../Strategist/Strategist.h"
```

Comment:
Strategist is already included in this file, no reason I can think of to include it twice.